app/src/main/java/edu/nextstep/camp/calculator/MainActivity.kt

```diff
@@ -4,65 +4,54 @@ import android.os.Bundle
 import android.widget.Toast
 import androidx.appcompat.app.AppCompatActivity
 import edu.nextstep.camp.calculator.databinding.ActivityMainBinding
-import edu.nextstep.camp.calculator.domain.Expression
 
-class MainActivity : AppCompatActivity() {
+class MainActivity : AppCompatActivity(), MainContract.View {
     private lateinit var binding: ActivityMainBinding
-    private var expression = Expression.EMPTY
+    private lateinit var presenter: MainPresenter
```

Comment:
View에서는 Presenter를 구현 타입이 아닌 인터페이스 타입으로 가지고 있어야 합니다. 
```suggestion
    private lateinit var presenter: MainContract.Presenter
```

Reply:
네 수정하였습니다!